app/src/main/res/values/strings.xml

```diff
@@ -348,33 +348,33 @@
     <string name="delete_collection_deleting_collection">Deleting collection</string>
 
     <!-- JournalViewer -->
-    <string name="journal_entries_list_empty">Collection is empty.\nMaybe it\'s still syncing?</string>
-    <string name="journal_entries_loading">Loading change log entries...</string>
+    <string name="journal_entries_list_empty">The collection is empty.\n(Maybe it\'s still syncing?)</string>
+    <string name="journal_entries_loading">Loading journal entries…</string>
```

Comment:
@tasn Is it "journal entries", or "changelog entries…"?

Reply:
Used to be journal entries, now "changelog entries".

Reply:
The app currently supports both etesync v1 and etesync v2. In EteSync v1 journal was used for both "changelog" and "collection". In v2 we separate the two. I updated all of the strings that are v2 only or common for both, but kept the strings that are v1 only as they were. I know it's not optimal, but given that the v1 strings are going away soon, it's not that big of a deal.

Reply:
I actually noticed there are a few strings I missed. :|
I'll have to have a proper sweep soon.